server/consumer.go

```diff
@@ -595,6 +598,9 @@ func (mset *stream) addConsumerWithAssignment(config *ConsumerConfig, oname stri
 			}
 		}
 	}
+	// Create ackMsgs queue now that we have a consumer name
+	o.ackMsgs = s.newIPQueue(fmt.Sprintf("[ACC:%s] consumer '%s' on stream '%s' ackMsgs", accName, o.name, mset.cfg.Name))
```

Comment:
This is totally fine, but for external logging I have been doing "'%s' > '%s' > '%s"" which is acc -> stream -> consumer

Reply:
The reason I chose this format, is that in the monitoring page the ">" is escaped and so does not look good. For a stream, it would be something like this:
```
"stream '$G' \u003e 'TEST' messages": {
      "pending": 123
}
```
where as with my version it is:
```
"[ACC:$G] stream 'TEST' messages": {
      "pending": 123
}
```
The queues' name is only reported in the monitoring page, so how it looks there is important.

Also, with current format, you can easily select queues for a given account, something like:
```
http://localhost:8223/ipqueuesz?queues=ACC:<account name here>
```
The `queues` is a filter that I check with `strings.Contains()` (so technically, even with `'acc' > 'stream'` that would work since I would find the account name in the string..

Reply:
Sounds good.